study.md

```diff
@@ -22,7 +22,7 @@ material. Instead, digest what you've read and repeat it in your own voice.
 In your own words, define ORM and explain why using an ORM is valuable.
 
 ```md
-<!-- your response here -->
+ORM is a technique that lets you store objects in a database for easy acess to them. Its valuable because you can acess those object without having to send and SQL code
```

Comment:
Object relational-mapping (ORM) is a method of representing arbitrary data as objects in a programming language.